test/CadlRanchProjects/payload/pageable/src/Generated/tspCodeModel.json

```diff
@@ -65,7 +65,7 @@
       "IsNullable": false
      },
      "IsRequired": false,
-     "IsReadOnly": true
+     "IsReadOnly": false
```

Comment:
do we know where this change come from?
It now does not have code change in our generated projects because if this only happens on those "list result" models, which will always have "output" usage, therefore the generator will never add setters to the properties. 
But I think we should know why we have this change, or the originate of this change, does it come from TCGC or the compiler or our emitter?

Reply:
> do we know where this change come from? It now does not have code change in our generated projects because if this only happens on those "list result" models, which will always have "output" usage, therefore the generator will never add setters to the properties. But I think we should know why we have this change, or the originate of this change, does it come from TCGC or the compiler or our emitter?

Chenjie told me it's the change in upstream azore core. That doesn't change the generated codes, though.

Reply:
The original PR: https://github.com/Azure/typespec-azure/pull/549